test/rootRdf.txt

```diff
@@ -0,0 +1,84 @@
+@prefix premis: <http://www.loc.gov/premis/rdf/v1#> .
+@prefix image: <http://www.modeshape.org/images/1.0> .
+@prefix ore: <http://www.openarchives.org/ore/terms/> .
+@prefix sv: <http://www.jcp.org/jcr/sv/1.0> .
+@prefix isl: <http://www.islandora.ca/ontologies/2016/02/28/isl/v1.0/> .
+@prefix test: <info:fedora/test/> .
+@prefix nt: <http://www.jcp.org/jcr/nt/1.0> .
+@prefix pcdm: <http://pcdm.org/models#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsi: <http://www.w3.org/2001/XMLSchema-instance> .
+@prefix mode: <http://www.modeshape.org/1.0> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix fedora: <http://fedora.info/definitions/v4/repository#> .
+@prefix nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo/v1.2/> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix ebucore: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix ldp: <http://www.w3.org/ns/ldp#> .
+@prefix xs: <http://www.w3.org/2001/XMLSchema> .
+@prefix fedoraconfig: <http://fedora.info/definitions/v4/config#> .
+@prefix mix: <http://www.jcp.org/jcr/mix/1.0> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+
+
+<http://localhost:8080/fcrepo/rest/> a ldp:RDFSource , ldp:Container , ldp:BasicContainer , <http://www.modeshape.org/1.0root> , <http://www.jcp.org/jcr/nt/1.0base> , <http://www.jcp.org/jcr/mix/1.0referenceable> ;
```

Comment:
Oh yeah totally, just used the old vagrant, Fedora 4.4.0 I think. I'll update it.